cmd/agent/common/misconfig/global.go

```diff
@@ -16,11 +16,24 @@ func ToLog() {
 	}
 }
 
+func ToLogProcessAgent() {
```

Comment:
Would be great instead of having a separate `ToProcessAgent()`, pass a value with the agent name to `ToLog`. It can actually be ignored for now since this is the only misconfig check. This would be simpler because:
- Same interface with `misconfig.ToLog`
- No need for separate `processAgentChecks` map and no separate `registerProcessAgentCheck`.

Eventually when we have more than one check here, we could change:
`var checks = map[string]checkFn{}` to `var checks = map[string]check{}

where `check` would contain `checkFn` as well as the list of agents it applies to.